src/modules/job-exec/exec.c

```diff
@@ -146,10 +146,17 @@ static void complete_cb (struct bulk_exec *exec, void *arg)
 static void output_cb (struct bulk_exec *exec, flux_subprocess_t *p,
                        const char *stream,
                        const char *data,
-                       int data_len,
+                       int len,
                        void *arg)
 {
     struct jobinfo *job = arg;
+    const char *cmd = flux_cmd_arg (flux_subprocess_get_cmd (p), 0);
+    jobinfo_log_output (job,
+                        flux_subprocess_rank (p),
+                        basename (cmd),
```

Comment:
Hm, yeah. I always just assumed `basename(3)` returned a pointer just past the last `/` in string, or if no `/` returned the same `char *` that was passed in. Sorry about the sloppiness here, but in this case maybe we can introduce our own version...

Reply:
Well it works fine here since `cmd` is a `const char *`.  Maybe add a comment if you think it's necessary, but I was just logging my epiphany and not requesting a change.